pkg/config/config.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/viper"
+)
+
+type Config struct{}
```

Comment:
really I was thinking in having field string NodeConfig inside this struct, fill it up in New, returning algo error there.
avoid the accesor function LoadNodeConfig and the error return for it.

Reply:
I don't think we want to do that. Viper actually has the capability to refresh configs if the file changes. In the future, we may want to have long running commands. If we do, this approach will allow us to always use the latest values.